Essentials/src/main/resources/messages.properties

```
@@ -265,8 +279,20 @@ discordErrorNoPrimary=You did not define a primary channel or your defined prima
 discordErrorNoPrimaryPerms=Your bot cannot speak in your primary channel, #{0}. Please make sure your bot has read and write permissions in all channels you wish to use.
 discordErrorNoToken=No token provided! Please follow the tutorial in the config in order to setup the plugin.
 discordErrorWebhook=An error occurred while sending messages to your console channel\! This was likely caused by accidentally deleting your console webhook. This can usually by fixed by ensuring your bot has the "Manage Webhooks" permission and running "/ess reload".
+discordLinkInvalidGroup=An invalid group, {0}, was provided for role: {1}. The following groups are available: {2}
```

Not really particularly fond of the use of punctuation in this message and similar right now, but I don't have a suitable suggestion right away that doesn't involve completely removing them (which may be more confusing).

---

Maybe this will work? You can keep the colon if you want
```suggestion
discordLinkInvalidGroup=Invalid group {0} was provided for role {1}. The following groups are available: {2}
```

---

Yeah I like that better